manifests/o/osara/osara/2022.1pre-951,52c55fb0/osara.osara.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: osara.osara
+PackageVersion: 2022.1pre-951,52c55fb0
+PackageLocale: en-US
+Publisher: Reaper Accessibility group

```

```suggestion
Publisher: James Teh
```
This must match control panel